src/js/tracks/text-track.js

```diff
@@ -20,6 +20,14 @@ import XHR from 'xhr';
  * @param {Track} track track to addcues to
  */
 const parseCues = function(srcContent, track) {
+  // Make sure that vttjs has loaded, otherwise, wait till it finished loading
+  // NOTE: this is only used for the alt/video.novtt.js build
+  if (typeof window.WebVTT !== 'function') {
+    return window.setTimeout(function() {
```

Comment:
Yes, it would. It should have a retry count, I guess.
Maybe changing it to send an event we can listen to is a better way of doing it, I'll see if I can do it quickly, I'd like to get this change in as quickly as possible.